app/dependencies.php

```diff
@@ -9,9 +9,8 @@
 $container = $app->getContainer();
 
 //Add my db connection to the dependency container
-$container['db'] = function ($container) {
-    $config = $container['config']['db'];
-    return new mysqli($config['host'], $config['user'], $config['pass'], $config['db_name']);
+$container['db'] = function ($container) use ($config) { //TODO get to grips with what the use is doing here...
```

Comment:
You can find the official documentation on [function() use () here](http://php.net/manual/en/functions.anonymous.php#example-160)